Dark Unit Test Failure

The removal of the `err` array is causing a dark unit test to fail. Specifically:

FAILED romancal/dark_current/tests/test_dark.py::test_dark_step_output_dark_file[WFI-WFI_IMAGE] - AttributeError: No such attribute (err) found in node

The `err` test lines simply need to be removed.

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -6,6 +6,11 @@ associations
 
 - Add FOV associations to the  code  [#931]
 
+dark
+----
+
+- Removed ``err`` array from dark current tests. [#938]
+
 general
 -------
 

--- a/romancal/dark_current/tests/test_dark.py
+++ b/romancal/dark_current/tests/test_dark.py
@@ -45,10 +45,8 @@ def test_dark_step_interface(instrument, exptype):
     assert result.data.shape == shape
     assert result.groupdq.shape == shape
     assert result.pixeldq.shape == shape[1:]
-    assert result.err.shape == shape
     assert result.meta.cal_step.dark == "COMPLETE"
     assert result.data.dtype == np.float32
-    assert result.err.dtype == np.float32
     assert result.pixeldq.dtype == np.uint32
     assert result.groupdq.dtype == np.uint8
 
@@ -126,7 +124,6 @@ def test_dark_step_output_dark_file(tmpdir, instrument, exptype):
     assert dark_out_file_model.validate() is None
     assert dark_out_file_model.data.shape == shape
     assert dark_out_file_model.dq.shape == shape[1:]
-    assert dark_out_file_model.err.shape == shape
 
 
 def create_ramp_and_dark(shape, instrument, exptype):

